tools/grift/grpc_plugins_generator.patch

```diff
@@ -0,0 +1,2417 @@
+From 0894590b5020c38106d4ebb2291994668c64f9dd Mon Sep 17 00:00:00 2001
```

Comment:
I think we cant submit it to upstream. Maybe we can fork apache thrift to
google/grift and apply these changes. grift will be the submodule.

On Tue, Aug 2, 2016 at 1:24 PM, Nicolas Noble <notifications@github.com>
wrote:

> In tools/grift/grpc_plugins_generator.patch
> <https://github.com/grpc/grpc/pull/7594#discussion_r73229914>:
>
> > @@ -0,0 +1,2417 @@
> > +From 0894590b5020c38106d4ebb2291994668c64f9dd Mon Sep 17 00:00:00 2001
>
> I don't think this is a bad interim solution, yet ideally, we would submit
> them upstream. It would quickly become un-maintainable however if this
> isn't pushed upstream, that is.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/grpc/grpc/pull/7594/files/dd7a2a35e23042413c232ea9f00a4b7dbbce5648#r73229914>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ASfuImYF97g5NVIdvBzHVvASY3R6zaAyks5qb6dhgaJpZM4JZPv_>
> .
>